src/pages/waiting-room-to-car/cat-a-mod2/components/vehicle-checks-question/vehicle-checks-question.html

```diff
@@ -8,7 +8,7 @@
                     okText="Submit"
                     cancelText="Cancel" 
                     [formControlName]="questionFieldName"
-                    (ionChange)="vehicleChecksQuestionChanged($event)"
+                    (ionChange)="safetyAndBalanceQuestionChanged($event)"
                     [selectOptions]="{cssClass:'mes-select vehicle-checks-question-select', enableBackdropDismiss:false}">
```

Comment:
is it worth renaming css classes for consistency?

Reply:
I said rename but it would effectively be cloning the vehicle-checks-question-select class..... not sure about this.